nasa/earthdata-download
# EDD-40: Adding github page

# Overview

### What is the feature?

Moves the download page from Earthdata Search to a github page.

### What is the Solution?

The Earthdata Search download page was recreated using a vanilla Vite starter. It's mainly comprised of html, scss, and a small browser detection library. Some simple linting is in place as well. Github actions are in place to build and deploy to Github Pages. Values for the download links and sizes for the various OS's have been added as Vite environment variables in the `.env` file.

### What areas of the application does this impact?

Everything will be contained on the `gh-pages` branch.

# Testing

### Reproduction steps

- **Environment for testing:** N/A
- **Collection to test with:** N/A

1. Visit the Github page
2. Confirm the primary download link matches your OS
3. Confirm the links are the latest download links



### Attachments

Please include relevant screenshots or files that would be helpful in reviewing and verifying this change.

# Checklist
